stable/spring-cloud-data-flow/values.yaml

```diff
@@ -103,3 +103,93 @@ features:
     enabled: true
   batch:
     enabled: true
+  monitoring:
+    enabled: false
+
+grafana:
+  service:
+    type: LoadBalancer
+  admin:
+    existingSecret: scdf-grafana-secret
+    userKey: admin-user
+    passwordKey: admin-password
+    defaultUsername: YWRtaW4=
+    defaultPassword: cGFzc3dvcmQ=
+  extraConfigmapMounts:
+   - name: scdf-grafana-ds-cm
+     mountPath: /etc/grafana/provisioning/datasources/datasources.yaml
+     subPath: datasources.yaml
+     configMap: scdf-grafana-ds-cm
+     readOnly: true
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+        - name: default
+          orgId: 1
+          folder:
+          type: file
+          disableDeletion: true
+          editable: false
+          options:
+            path: /var/lib/grafana/dashboards/default
+  dashboards:
+    default:
+      scdf-applications:
+        url: https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/src/grafana/prometheus/docker/grafana/dashboards/scdf-applications.json
```

Comment:
Should these links point to 2.3.0 tagged files instead?

Reply:
the dashboards don't seem to be release related, using master here would pickup any latest enhancements to them and one less version to maintain. @ctzolov may have other thoughts?